packages/amplify-cli/src/commands/upgrade.ts

```diff
@@ -1,25 +1,32 @@
-import { $TSContext, isPackaged, pathManager } from 'amplify-cli-core';
+import {
+  $TSContext, isPackaged, pathManager,
+} from 'amplify-cli-core';
 import fetch from 'node-fetch';
 import { gt } from 'semver';
 import * as path from 'path';
 import * as fs from 'fs-extra';
-import { oldVersionPath } from '../utils/win-constants';
 import chalk from 'chalk';
 import gunzip from 'gunzip-maybe';
 import tar from 'tar-fs';
 import ProgressBar from 'progress';
 import { pipeline } from 'stream';
 import { promisify } from 'util';
+import ora from 'ora';
+import execa from 'execa';
+import * as emoji from 'node-emoji';
```

Comment:
I only see the the `@types/node-emoji` devDependency. We would need to add `node-emoji` to `amplify-cli/package.json`. Although, is a whole new dependency really necessary for one emoji?

Reply:
good point, can replace with a text message.